### PR TITLE
Add upgrade centreon double run

### PR DESCRIPTION
**Pull request by psamecentreon**
## Description

This is a documentation to implement the migration of Centreon between different OS by the double-run method

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [X] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.04.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+Carry out the complete installation of the new Centreon platform (central + database in our case) on the new servers by following the [following procedure](../installation/introduction.md).
+
+## Backup before migration
+
+Backup the fresh databases.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.04.md:56]: similar remarks as for version 21.10

File: versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.10.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+Carry out the complete installation of the new Centreon platform (central + database in our case) on the new servers by following the [following procedure](../installation/introduction.md).
+
+## Backup before migration
+
+Backup the fresh databases.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.10.md:56]: similar remarks as for version 21.10

File: versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-21.04.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+Carry out the complete installation of the new Centreon platform (central + database in our case) on the new servers by following the [following procedure](../installation/introduction.md).
+
+## Backup before migration
+
+Backup the fresh databases.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline versioned_docs/version-22.04/upgrade/double-run/upgrade-double-run-from-21.04.md:56]: similar remarks as for version 21.10

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.04.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+Réaliser l'installation complète de la nouvelle plateforme Centreon (central + bdd dans notre cas) sur les nouveaux serveurs en suivant la [procédure suivante](../installation/introduction.md).
+
+## Sauvegarde avant migration
+
+Sauvegarder les bases vierges.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.04.md:56]: similar remarks as for version 21.10 EN

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.10.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+Réaliser l'installation complète de la nouvelle plateforme Centreon (central + bdd dans notre cas) sur les nouveaux serveurs en suivant la [procédure suivante](../installation/introduction.md).
+
+## Sauvegarde avant migration
+
+Sauvegarder les bases vierges.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.10.md:56]: similar remarks as for version 21.10 EN

Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-20.10.md:56]: similar remarks as for version 21.10 EN

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-21.04.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+Réaliser l'installation complète de la nouvelle plateforme Centreon (central + bdd dans notre cas) sur les nouveaux serveurs en suivant la [procédure suivante](../installation/introduction.md).
+
+## Sauvegarde avant migration
+
+Sauvegarder les bases vierges.
+
+- **@NEW-DB@** :
+
+```bash
+mysqldump -u root -p <rootPassword> centreon > db_centreon_fresh_install_`date +%Y%m%d-%H%M`.sql
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/upgrade/double-run/upgrade-double-run-from-21.04.md:56]: similar remarks as for version 21.10 EN

**Review by cedricmeschin — CHANGES_REQUESTED** [review]

**Review by cedricmeschin — APPROVED** [review]

**Comment by cg-tw** [comment]
Internal doc

**Pull request closed without merging**